lib/interfaces/ISequelizeConfig.ts

```diff
@@ -1,7 +1,6 @@
-import {Options} from 'sequelize';
+import { Options } from 'sequelize';
 
```

Comment:
In general: can you revert these style changes everywhere? We're aren't using spaces on the start and the end in `{}` Thank you :)

Reply:
ok not problem, i will disable my formatter and create another pull request. is that ok?

Reply:
The changes will take place automatically on this PR already - So no need for creating another one :)